napari/layers/labels/labels.py

```diff
@@ -1049,6 +1049,14 @@ def _raw_to_displayed(
         if sliced_labels is None:
             sliced_labels = labels[data_slice]
 
+        if sliced_labels.dtype.itemsize <= 2:
+            self.colormap._map_to_gpu(sliced_labels)
```

Comment:
It is strange to not use the return type here? If it's expected (eg to create a cache?), then please add a comment about why it's not used?

Reply:
it is mmissed return